src/app/request-copy/email-request-copy/email-request-copy.component.html

```diff
@@ -12,18 +13,46 @@
     <label for="message" class="form-label">{{ 'grant-deny-request-copy.email.message' | translate }}</label>
     <textarea class="form-control" id="message" rows="8" [(ngModel)]="message" name="message"></textarea>
   </div>
+  <!-- Display access periods if more than one was bound to input. The parent component (grant-request-copy)
+       sends an empty list if the feature is not enabled or applicable to this request. -->
+  @if (hasValue(validAccessPeriods$ | async) && (validAccessPeriods$ | async).length > 0) {
+    <div class="form-group">
+      <label for="accessPeriod">{{ 'grant-request-copy.access-period.header' | translate }}</label>
+      <div ngbDropdown class="d-block">
+        <!-- Show current selected access period (defaults to first in array) -->
+        <button
+          class="btn btn-outline-primary playlist"
+          id="accessPeriod"
+          ngbDropdownToggle
+        > {{ 'grant-request-copy.access-period.' + this.accessPeriod | translate }}
+        </button>
+        <!-- Access period dropdown -->
+        <div ngbDropdownMenu aria-labelledby="accessPeriod">
+          @for (accessPeriod of (validAccessPeriods$ | async); track accessPeriod) {
+            <button
```

Comment:
This dropdown looks very weird in the `custom` theme:
![20250327-141128](https://github.com/user-attachments/assets/1977d9fd-19c0-4a1a-a7ee-e9e21e4dde25)

...vs. in the `dspace` theme:
![20250327-141325](https://github.com/user-attachments/assets/d6a4838e-c0a2-4855-844b-7776def3e509)

Reply:
i hadn't seen this and it just followed the same pattern of ngbDropdownMenu used elsewhere but I'm trying a few things out, please let me know if you still see that